### ISSUE TITLE
Add animation-play-state

### ISSUE THREAD

**Issue by liamcurry** [mirisuzanne/compass-animation#2]
The [`animation-play-state`](http://dev.w3.org/csswg/css3-animations/#the-animation-play-state-property-) property is missing.

It'd look something like this:

``` scss
$default-animation-play-state: running;

// ---------------------------------------------------------------------------
@mixin animation-play-state($state-1: $default-animation-play-state, $state-2: false, $state-3: false, $state-4: false, $state-5: false, $state-6: false, $state-7: false, $state-8: false, $state-9: false, $state-10: false) {
  $state: compact($state-1, $state-2, $state-3, $state-4, $state-5, $state-6, $state-7, $state-8, $state-9, $state-10);
  @include animation-support(animation-play-state, $state); }
```

**Comment by mirisuzanne** [comment]
Thanks! Care to submit that as a patch?

You'll want to update the +animation() default with this additional argument as well (in `$animation-1: -compass-space-list(...)`)

**Comment by mirisuzanne** [comment]
closed by a054974eda04b321e330d84c92339bb165807d46
